### DISCUSSION TITLE
Adding slider control or stop toggle

### DISCUSSION THREAD

**Discussion by fargoboy1** [aarons22/homebridge-bond#155] [Ideas]
Just a quick question/request for shades. With bond app I’m given an open, close, stop buttons. With the bond plug-in I’m given choice of all the way open or all the way shut. I don’t have the ability to stop the shade part way. Is it possible to change to a true slider control with increments of opening or closing? Or toggle to stop shade part way? 
I think the plug in great otherwise! Controlling 3 fans and 4 hurricane shades.

**Comment by chrismerck** [comment]
On the Bond side, we have a Position feature with SetPosition available on Bond Bridge Pro, currently that feature is in beta and for Somfy RTS only.

**Reply by fargoboy1** [reply]
So no go for the basic bond bridge? No ability to add a stop toggle?

**Reply by chrismerck** [reply]
Oh you can still send the Hold action on BD-1000, it's just that only with the upgraded hardware can we accurately time the transmissions and permit "true slider" control.

**Comment by fargoboy1** [comment]
So I have the BD-1000, but am not able to send a hold action. My shades are hurricane shades using the Dooya remotes and motors. Am I missing something? I can open, close, but no way to hold with the plug in. The bond app allows me to hold but I prefer to use HomeKit if possible. It’s not the end of the world, but I’m just curious. 
Thanks!

**Comment by webharvest** [comment]
I wanted to see if we are able to add the hold/stop command to this plugin?  It sounds like Bond can support the hold command but the plug-in doesn’t have a option for it.

**Reply by dscheck** [reply]
I have the same issue using Bond with Homebridge and StormSmart shutters.. I would like to have the stop/hold button added to the plugin for HomeKit please..

**Comment by gitpatrickhub** [comment]
+1

would like to add the stop/hold feature for Bond Bridge (non-pro version). Willing to donate to whomever for support.

**Reply by gitpatrickhub** [reply]
Stop action exists in bondapi.ts but is not part of the shadesaccessory.ts. 

If this action was implemented then an Apple Shortcut could be used to open or close at approximate % by using Open “wait N seconds” then Stop. It wouldn’t be exact but close enough for me.
